microsoft/PowerToys
# [KBM] Open Remap Shortcuts/Remap Keyboard window on the same monitor as Settings.

## Summary of the Pull Request

_Remap Shortcuts_/_Remap Keyboard_ window opens on the same monitor as Settings.

## PR Checklist
* [x] Applies to #2832

## Info on Pull Request

* Moved `GetAllMonitorRects` and `GetAllMonitorInfo` to avoid code duplication.
* Changed window opening position.

## Validation Steps Performed

Move Settings window on a secondary monitor, click _Remap a key_/_Remap a shortcut_, verify that window was opened on the same monitor as Settings.
